### PR TITLE
Add StrongName assembly Signing -> version 1.3.0

### PR DESCRIPTION
**Pull request by jgaulon**
I would like to use this library in my project, but I need it signed.
Can you please check my Pull Request?
Thanks in advance.

### CODE REVIEW

**Comment by jgaulon** [comment]
@HakanL can you please help here? why is PACKAGE_VERSION=1.2.${GITHUB_RUN_NUMBER} and not PACKAGE_VERSION=1.**3**.${GITHUB_RUN_NUMBER} like in version.json?

Thanks.

**Comment by HakanL** [comment]
@jgaulon there was a bug in the action script. Please rebase to master and re-try.

File: src/PcapngUtils/PcapngUtils.csproj [hunk 1]
```diff
@@ -5,20 +5,22 @@
     <RootNamespace>Haukcode.PcapngUtils</RootNamespace>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
-    <Version>0.999-debug</Version>
```
Comment (HakanL) [inline src/PcapngUtils/PcapngUtils.csproj]: We shouldn't change this version, it's used when building locally, so it should be version 0.999

File: src/PcapngUtils/PcapngUtils.csproj [hunk 2]
```diff
@@ -5,20 +5,22 @@
...
     <Copyright>Copyright © Janusz Ryrych</Copyright>
     <PackageProjectUrl></PackageProjectUrl>
     <RepositoryUrl>https://github.com/HakanL/PcapngUtils</RepositoryUrl>
     <RepositoryType>GIT</RepositoryType>
-    <PackageReleaseNotes>1.2.0 Changed Seconds/Microseconds to uint. Renamed enchanted to enhanced
+    <PackageReleaseNotes>1.3.0 Assembly signing 1.2.0 Changed Seconds/Microseconds to uint. Renamed enchanted to enhanced
 1.1.0 Added new methods to get next packet as an alternative to streaming via callback event. Also added ways to add streams to the constructors.
 1.0.8 Converted to .NET Standard 2.0 and removed NUnit dependencies</PackageReleaseNotes>
     <PackageId>Haukcode.PcapngUtils</PackageId>
-    <AssemblyVersion>0.999.0.0</AssemblyVersion>
```
Comment (HakanL) [inline src/PcapngUtils/PcapngUtils.csproj:20]: See above, revert this.

File: src/PcapngUtils/PcapngUtils.csproj [hunk 3]
```diff
@@ -5,20 +5,22 @@
...
-    <Version>0.999-debug</Version>
+    <Version>1.3.0</Version>
     <Description>C# full managed implementation Pcap/PcapNG file format</Description>
     <Company></Company>
     <Authors>Janusz Ryrych, Hakan Lindestaf</Authors>
     <Copyright>Copyright © Janusz Ryrych</Copyright>
     <PackageProjectUrl></PackageProjectUrl>
     <RepositoryUrl>https://github.com/HakanL/PcapngUtils</RepositoryUrl>
     <RepositoryType>GIT</RepositoryType>
-    <PackageReleaseNotes>1.2.0 Changed Seconds/Microseconds to uint. Renamed enchanted to enhanced
```
Comment (HakanL) [inline src/PcapngUtils/PcapngUtils.csproj:16]: Add a newline before 1.2.0

**Review by HakanL — CHANGES_REQUESTED** [review]

File: src/PcapngUtils/PcapngUtils.csproj (hunk 3 above)
Comment (jgaulon) [reply src/PcapngUtils/PcapngUtils.csproj:16]: done

File: src/PcapngUtils/PcapngUtils.csproj (hunk 1 above)
Comment (jgaulon) [reply src/PcapngUtils/PcapngUtils.csproj]: done

File: src/PcapngUtils/PcapngUtils.csproj (hunk 2 above)
Comment (jgaulon) [reply src/PcapngUtils/PcapngUtils.csproj:20]: done

**Comment by HakanL** [comment]
Thanks! It seems that the action script still didn't work 100%, let me see what's going on.

**Comment by jgaulon** [comment]
Thanks for your support on this!

**Comment by HakanL** [comment]
@jgaulon Can you try to re-base again?

**Comment by jgaulon** [comment]
> @jgaulon Can you try to re-base again?

done

**Comment by HakanL** [comment]
Thanks, for some reason it's still using 1.2, but the files look correct. Not sure why it would use the version.json file froim the master. But it may correct itself when merged, so I'll go ahead and merge. Thank you for your contribution!

**Pull request merged**